include/ofxPixels.h

```diff
@@ -78,7 +78,7 @@ typedef struct OfxRGBColourF {
 
 /** @brief Defines a double precision floating point component RGB pixel */
 typedef struct OfxRGBColourD {
-  double r, g, b, a;
+  double r, g, b;
```

Comment:
Note that this struct is not used anywhere AFAIK, it seems to be here for convenience.
It was obviously a typo in the original version of the include (OfxRGBColourB OfxRGBColourS OfxRGBColourF were OK, as were all the RGBA versions). I noticed that when I wrote a function that actually took an RGB triplet as argument.